test/jit/test_graph_rewrite_passes.py

```diff
@@ -4,58 +4,149 @@
 import torch
 import torch._C
 from torch.testing import FileCheck
+from typing import Callable, Dict
+from os import environ
+import unittest
 
+class FunctionalConv2d(torch.nn.Module):
+    def __init__(self, weight: torch.Tensor, bias: torch.Tensor) -> None:
+        super(FunctionalConv2d, self).__init__()
+        self.weight = weight
+        self.bias = bias
+
+    def forward(self, x: torch.Tensor) -> torch.Tensor:
+        x = torch.nn.functional.conv2d(
+            input=x.unsqueeze(dim=0),
```

Comment:
(to execute the vulkan model) remove the `unsqueeze` here since `unsqueeze` isn't implemented for Vulkan tensors yet.

```
input=x,
```